tests/installation/disable_grub_timeout.pm

```diff
@@ -15,6 +15,7 @@ use strict;
 use warnings;
 use base "y2logsstep";
 use testapi;
+use utils;
 
 sub run {
```

Comment:
I would save the hotkey in a var at that point:

```
if (check_var('DISTRI', 'sle') && sle_version_at_least('12-SP2') || check_var('DISTRI', 'opensuse')) {
  var bootloader_hotkey = 'alt-r';
}
else {
  var bootloader_hotkey = 'alt-t';
}
```

Notice: && has precedence over ||, so you don't need extra parentheses.

Reply:
weird , I updated my branch with simuliar bit simpler change .. and here isnt update visible

Reply:
Yes, I also have a broken PR for that reason. It seems that github observer for new push'es to a branch is currently broken.

I had closed my old PR and did the PR again. It did work so. Could you try?